doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,159 @@
+To enable it, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_BINARY_LOGGING`.
+
+**Optional:** You can increase size of ring buffer that temporarily stores the trace logs.
+   * To do this, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_BUFFER_SIZE`
+   * This can prevent losing some of the logs in demanding scenarios --NOTE: LOOK INTO THIS - GIVE A SPECIFIC EXAMPLE?
+
+
+Trace logs using UART (default)
+===============================
+
+Trace logs using the UART are enabled by default.
+Below are more details on how to configure trace logs using the UART.
+
+
+1. This is enabled by default. It can be set with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_UART_LOGGING`.
+
+#. Configure which UART device to use with :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME`.
+   This is optional, as the default ``UART_1`` will be used if no other UART device is configured.
+
+#. Configure ``UART_1`` to be connected to the onboard J-Link instead of ``UART_0``, by extending the DTS overlay file for the selected board with the following:
+
+    .. code-block:: devicetree
+
+      &uart1 {
+         tx-pin = <6>;
+         rx-pin = <8>;
+         rts-pin = <5>;
+         cts-pin = <7>;
+      };
+
+      &uart0 {
+         rx-pin = <33>;
+         tx-pin = <34>;
+         /delete-property/ rts-pin;
+         /delete-property/ cts-pin;
+      };
+
+   .. note::
+      This will remove access to the application logs.
+
+
+Optional configuration
+-----------------------
+You can also increase the UART throughput by changing baudrate.
+Some of the trace logs will be dropped if the throughput is too low.
+By default, the UART baudrate is set to 115200.
+To increase the baudrate to ``1000000``, add the ``current-speed = <1000000>;`` property to the ``uart1`` node in the DTS overlay file.
+
+
+
+Trace logs using native USB
+===========================
+
+Trace logs can also be configured using a native USB.
+
+
+1. To configure, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_USB_CDC_LOGGING`.
+   This also enables the necessary USB Kconfig options.
+
+#. Configure which USB device to use with :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME`.
+   This is optional, as the default ``CDC_ACM_0`` will be used if no other USB device is configured.
+
+#. Create a USB CDC ACM instance that will be used for printing ZBOSS trace logs by extending the DTS overlay file for the selected board with the following:
+
+   .. code-block:: devicetree
+
+      &zephyr_udc0 {
+         cdc_acm_uart0: cdc_acm_uart0 {
+            compatible = "zephyr,cdc-acm-uart";
+            label = "CDC_ACM_0";
+         };
+      };
+
+   .. note::
+      For the ZBOSS trace logs to be printed correctly through the USB, it is recommended to not use the USB autosuspend.
+
+
+
+Collecting log files
+********************
+
+To collect and view the log files, you will need the correct software, and to be able to find the correct COM port.
+
+* Which software to use:
```

Comment:
Change this to "Requirements" subhead, and move it further up in the doc?
Also include a table of which DKs are required and compatible for the trace logs?